terminus-ui/select/src/select.component.spec.ts

```diff
@@ -1012,6 +1056,26 @@ describe(`TsSelectComponent`, function() {
           expect(instance.autocompleteFormControl.value).toEqual(['Arkansas']);
         });
 
+
+        test(`should set single value 2`, () => {
+          const fixture = createComponent(testComponents.SeededAutocompleteWithFormatFn);
+          fixture.componentInstance.allowMultiple = false;
+          fixture.componentInstance.keepOpen = false;
+          fixture.componentInstance.myCtrl.setValue([{ name: 'Florida', population: '20.27M'}]);
+          fixture.detectChanges();
```

Comment:
Add an expect here to make sure it was selected correctly and the reset changes the value.